sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/object.scala

```diff
@@ -83,6 +83,16 @@ case class SerializeFromObject(
     child: LogicalPlan) extends UnaryNode with ObjectConsumer {
 
   override def output: Seq[Attribute] = serializer.map(_.toAttribute)
+
+  // We can't estimate the size of ObjectType. We implement statistics here to avoid
+  // directly estimate any child plan which produces domain objects as output.
+  override def statistics: Statistics = {
+    if (child.output.find(_.dataType.isInstanceOf[ObjectType]).isDefined) {
+      Statistics(sizeInBytes = Long.MaxValue)
```

Comment:
Not related to this PR, but why do we need to store the `sizeInBytes` not `numRows`? If we have the `numRows` info, we can still estimate the output size for this operator right?